### DISCUSSION TITLE
Disscussionに慣れるためのdiscussion

### DISCUSSION THREAD

**Discussion by noborus** [pgsql-jp/jpug-doc#2455] [General]
お気軽に試してみてください

**Comment by koizumistr** [comment]
カテゴリーにpollとかあるので試してみたいなぁ、とも思いますが、そもそもどうやるんだか分らなかった、、、
少なくとも、この項目をpollに変えてやろうかと思ったが、選択肢に出てこないですね、、、、

**Reply by noborus** [reply]
pollはissueからのコンバートか最初に選択する必要がありそうですね。やってみましょうか。

**Reply by noborus** [reply]
やってみました。コンバートではうまくいかなかった...
https://github.com/pgsql-jp/jpug-doc/discussions/2473#discussion-4564691
